### PR TITLE
Upgraded to node 7.8 and latest packages

### PR DESCRIPTION
**Pull request by glenna-m**
Rolling in previous closed PR change to support simple boolean values our our RDF

### CODE REVIEW

File: bin/ttl-to-json [hunk 1]
```diff
@@ -103,8 +105,7 @@ fileAccessible(cmds.ttlfile);
 if (!_.isUndefined(cmds.frame)) {
    if (_.isNull(cmds.frame)) {
        console.error("Frame argument requires a frame file!");
-       help();
-       process.exit(1);
+       help(1);
```
Comment (dmtaub) [inline bin/ttl-to-json]: if help always exits, maybe change to "helpAndExit(1)" or use a variable exitStatus (or a comment) so it's clearer what you're passing 1 for.

File: bin/ttl-to-json [hunk 2]
```diff
@@ -145,7 +146,7 @@ Promise.all(readFiles).then(fileContent => {
      // from the original json input on a round trip translation. 
      var filterBnodeAttrs = [ 'id', '_id', '@id' ];
 
-     ttlJsonLd.rdfToJsonLd(graph, frameText, filterBnodeAttrs).then(json => { 
+     TtlJsonLd.rdfToJsonLd(graph, frameText, filterBnodeAttrs).then(json => {
```
Comment (dmtaub) [inline bin/ttl-to-json]: I like this convention of using capital letters to indicate imported objects, but there is a (not necessarily contradictory) convention to use it for class names . Whatever the conventions we go with, let's try to document them in the team-docs repo -- maybe a CONTRIBUTING.md file

**Comment by dmtaub** [comment]
LGTM

File: bin/ttl-to-json (hunk 1 above)
Comment (dbooth-boston) [inline bin/ttl-to-json]: Glenna, were you going to rename this function?

Comment (glenna-m) [inline bin/ttl-to-json]: Done in the last commit.  Good suggestion.

**Comment by glenna-m** [comment]
Daniel asked me to document the areas where I think test coverage should be improved.  I did an Istanbul coverage run which shows pretty clearly where the gaps are. 
screen shot 2017-04-17 at 11 08 43 am

**Comment by glenna-m** [comment]
David, do you want me to spend additional time documenting the areas that need testing beyond the Istanbul analysis, or just add an issue?

**Comment by dbooth-boston** [comment]
Just log the need for more testing as an issue on our main issues list, and let's not do more work on it now.  Ideally we should have more tests, but we have much higher priorities right now.

**Comment by glenna-m** [comment]
Issue added: https://github.com/rdf-pipeline/noflo-rdf-pipeline/issues/77

**Pull request merged**